doc/classes/Skeleton3D.xml

```diff
@@ -47,14 +47,14 @@
 		<method name="force_update_all_bone_transforms" deprecated="This method should only be called internally.">
 			<return type="void" />
 			<description>
-				Force updates the bone transforms/poses for all bones in the skeleton.
+				Force recalculate dirty bone transforms/poses for all bones in the skeleton.
```

Comment:
I think changing this description is more confusing.

My understanding is `force` implies that **all** bones are updated, not only dirty ones. So the old explanation is more accurate.

Perhaps something like:
`Forcefully recalculate all bone transforms/poses for all bones in the skeleton.`

or if it really does only update dirty bones,
`Recalculate only dirty bone transforms in the skeleton.`